pkgs/os-specific/linux/tuxedo-keyboard/default.nix

```diff
@@ -2,15 +2,17 @@
 
 stdenv.mkDerivation rec {
   pname = "tuxedo-keyboard-${kernel.version}";
-  version = "3.2.5";
+  version = "unstable-2023-06-19";
```

Comment:
nit: Can you add a comment here that we only use master, because the most recent version is not compatible to 6.4?

Reply:
will add a comment!